glyphs-reader/src/font.rs

```diff
@@ -2590,6 +2593,12 @@ impl Font {
         &self.masters[self.default_master_idx]
     }
 
+    pub fn instance(&self, master: &FontMaster) -> Option<&Instance> {
+        self.instances
+            .iter()
+            .find(|i| i.active && i.type_ == InstanceType::Variable && i.name == master.name)
+    }
+
```

Comment:
if this is only being used to find the default instance, maybe just a `default_instance` fn that didn't need to have the master passed in would be easier?

Reply:
I _think_ there are going to be mroe cases